tests/testthat/test-lexical_pmentities.R

```diff
@@ -0,0 +1,13 @@
+context("lexical_pmentities")
+
+test_that("lexical_pmentities", {
+  tt <- lexical_pmentities(pmid = 1)
+  
+  expect_is(tt, "data.frame")
+  expect_is(tt$token, "character")
+  expect_equal(length(tt$token), 4)
+  expect_is(tt$start, "integer")
+  expect_equal(tt$start, c(0, 7, 45, 66))
```

Comment:
@steven2249 running locally, the test on line 10 fails for me. check that one - I'm guessing that the value returned can vary through time - if so, remove that test, or change it to something that will pass no matter when it's called.